src/api/firebase.js

```diff
@@ -0,0 +1,28 @@
+import * as firebase from 'firebase/app';
+import 'firebase/firestore';
+import 'firebase/firebase-auth';
+import 'firebase/analytics';
+
+// Your web app's Firebase configuration
+var firebaseConfig = {
+  apiKey: 'AIzaSyCr9W670NPUv2MmP_wH_5-5cTOpJ_01WlM',
```

Comment:
to make it more professional use the dotenv ".env" to hide the apiKey

Reply:
this and other sensitive information

Reply:
Okay